charts/sonarqube/README.md

```diff
@@ -29,6 +29,14 @@ The above command deploys SonarQube on the Kubernetes cluster in the default con
 
 The default login is admin/admin.
 
+## Installing the LTS chart
```

Comment:
Similar as for sonarqube-dce:
```suggestion
## Installing the SonarQube 9.9 LTS chart
```

Note that the reference to this section would then need to be adjusted in the introduction.

Reply:
i think you are right, for the sake of the history version, it might be interesting to have it. amending